components/ntp_sponsored_images/browser/regional_component_data.cc

```diff
@@ -0,0 +1,754 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/ntp_sponsored_images/browser/regional_component_data.h"
+
+// This list should be synced with the list of generateNTPSponsoredImages.js
```

Comment:
can we put json files in a grd file instead and then load these as resources for c++ and js so we're using the same code for both?

Reply:
this is a lot to put in a .cc file anyway

Reply:
I'm not sure if `generateNTPSponsoredImages.js` is in this repo or the CRX-generation repo but it is not a chromium / webui file, so won't be able to read from resource. But I'm sure it can read from a separate file (if it's in the same repo!) - @simonhong @bridiver

Reply:
It's a different repo; since they're not likely to change often, I personally don't think it's worth wiring it up. But brave-crx-packager could generate contents of `regional_data` in a copy/paste-able format